Source/Blazorise/Components/NumericPicker/NumericPicker.razor.cs

```diff
@@ -86,7 +103,7 @@ public override async Task SetParametersAsync( ParameterView parameters )
                 {
                     ExecuteAfterRender( async () => await JSModule.UpdateOptions( ElementRef, ElementId, new
```

Comment:
Is decimalsChanged == false enough when `Integer`? I see other Decimal related configuration. Or would those not make sense to check for `integer`?

Reply:
They don't make sense if the type is Integer. That's why I ignore it.